examples/apps/generic_client/generic_client.py

```diff
@@ -178,6 +180,31 @@ def _lookup_first_worker(worker_registry, jrpc_req_id):
     return worker_id
 
 
+def _do_ias_verification(worker_obj):
+    # Do ias verfication on proof data if it exists
+    # Proof data exists in SGX hardware mode.
+    if not worker_obj.proof_data:
+        logger.info("Proof data is empty. " +
+                    "Skipping verification of attestation report")
+    else:
+        # Construct enclave signup info json
+        enclave_info = {
+            'verifying_key': worker_obj.verification_key,
+            'encryption_key': worker_obj.encryption_key,
+            'proof_data': worker_obj.proof_data,
+            'enclave_persistent_id': ''
+        }
+
+        logger.info("Perform verification of attestation report")
+        verify_report_status = attestation_util.verify_attestation_report(
+            enclave_info)
```

Comment:
Client needs to check if MRENCLAVE value of worker matches with proof data before submitting work orders. For that client needs to read MRENCLAVE value of worker from config file.

Reply:
Should client has to do it explicitly? Can't this thing done in verify_attestation_report API? Which config has the data about MRENCLAVE?

Reply:
verify_attestation_report API is probably a better place for verifying MRENCLAVE value since it can be more broadly reused.
We also should  consider verifying a) WPE MRENCLAVE (2nd value) b) MRSIGNER.
It makes sense to merge this PR and then create a new PR that will
- modify verify_attestation_report() to accept MRENCLAVE (singleton or KME), MRENCLAVE of 
  WPE, MRSIGNER. If any of the values is not provided (null or empty string), the function 
  should not verify it
- modify _do_IAS_verification() function to pass the new additional parameters. In case of the 
  generic client these parameters can be accepted on the command line or in a config file 
  (the config file name is provided as a command line parameter)